di_website/visualisation/models.py

```diff
@@ -189,6 +191,62 @@ class RawCodePage(CodePageMixin, Page):
     class Meta:
         verbose_name = 'Raw Code Snippet'
 
+class PivotTableRowHighlight(Orderable):
+    HIGHLIGHT_CONDITION = [
+        ('lt', 'Less Than'),
+        ('gt', 'Greater Than'),
+        ('eq', 'Equals'),
+        ('lte', 'Less Than or Equal'),
+        ('gte', 'Greater Than or Equal'),
+    ]
+
+    RED = '#E35335'
+    BLUE = '#A2A2D0'
+    PINK = '#ffb3b3'
+    YELLOW = '#f6a90f'
+    ORANGE = ' #eba832'
+    PURPLE = '#E6E6FA'
+    GREEN = '#2D5A27'
+    COLOUR_CHOICES = (
+    (RED, 'Red'),
+    (BLUE, 'Blue'),
+    (PINK, 'Pink'),
+    (YELLOW, 'Yellow'),
+    (ORANGE, 'Orange'),
+    (PURPLE, 'Purple'),
+    (GREEN, 'Green')
+    )
```

Comment:
Don't limit the colour ... leave it as a blank field... give it the default as I requested in the instructions. I know these peeps ... they're too fickle and I'd rather not have to code anymore just to accommodate them. If you're set on these colours, add them in the `help_text` as the suggested options

Reply:
I was thinking that if I left it blank, there would be room for error, felt safer knowing the color before hand

Reply:
It's manageable error, won't break anything & the flexibility it provides is worth it ... I'd rather not have comms or Bill come to me every few weeks asking me to add a new colour.

Reply:
Okay

Reply:
If you really want to be on the safe side, you can provide a link to a hex colour resource like this one https://hexcolorcodes.org/ in the help_text

Reply:
Otherwise I trust the people using this to know their way around HTML colours.